### PR TITLE
[NUI] fix bug when setting desired size in ImageView

### PR DESCRIPTION
**Pull request by tscholb**
before, ImageView calculate desired size using GetOriginalImageSize()
but this api does not consider rotate value
so i changed it

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 1]
```diff
@@ -1246,28 +1246,13 @@ private void UpdateImage(int key, PropertyValue value)
...
-                      adjustedDesiredHeight = imageSize.Height * _desired_width / imageSize.Width;
-                  }
-                  else
-                  {
-                      adjustedDesiredWidth = imageSize.Width * _desired_height/ imageSize.Height;
-                      adjustedDesiredHeight = _desired_height;
-                  }
-
-                  PropertyValue returnWidth = new PropertyValue(adjustedDesiredWidth);
+                  Size2D desiredSize = new Size2D(_desired_width,_desired_height);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:1249]: Could you check warning message?
Check warning on line 1249 in src/Tizen.NUI/src/public/BaseComponents/ImageView.cs
Check warning on line 1250 in src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

**Comment by rabbitfor** [comment]
@tscholb Please check huiyu's comments!

**Comment by dongsug-song** [comment]
@tscholb 
Hello,
please resolve the conflicts. otherwise please upload new PR and close this PR.
thank you.

**Comment by tscholb** [comment]
i will create new PR.

**Pull request closed without merging**